src/java/test/org/apache/zookeeper/server/quorum/CommitProcessorConcurrencyTest.java

```diff
@@ -106,6 +106,7 @@ public void shutdown() {
 
                 @Override
                 public void notifyStopping(String threadName, int errorCode) {
+                    Assert.fail("Commit processor crashed " + errorCode);
```

Comment:
My first commit failed as I used lamba so I changed it back to Java 7 style. I forgot to change the commit message when I submit the second one. I will see how to amend that.